sound/soc/sof/Kconfig

```diff
@@ -188,13 +188,24 @@ config SND_SOC_SOF_DEBUG_ENABLE_FIRMWARE_TRACE
 	  If unsure, select "N".
 
 config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
-	bool "SOF enable IPC flood test"
+	tristate "SOF enable IPC flood test"
+	select SND_SOC_SOF_CLIENT
 	help
-	  This option enables the IPC flood test which can be used to flood
-	  the DSP with test IPCs and gather stats about response times.
+	  This option enables a separate client device for IPC flood test
+	  which can be used to flood the DSP with test IPCs and gather stats
+	  about response times.
 	  Say Y if you want to enable IPC flood test.
 	  If unsure, select "N".
 
+if SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
+config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST_NUM
+	int "Number of IPC flood test clients"
+	range 1 32
+	default 1
```

Comment:
I can't recall why we said 1 is the default. If we want to test IPC concurrency 2 seems like a natural number, no?

Reply:
I don't want to introduce any functionality change. The original code provided one IPC flood test, we do the same by default but later we can change the default.